tests/e2e/Services/Databases/DatabasesBase.php

```
@@ -5690,6 +5699,15 @@ public function testDecrementAttribute(): void
             'x-appwrite-project' => $this->getProject()['$id'],
         ]), ['value' => 'not-a-number']);
         $this->assertEquals(400, $typeErr['headers']['status-code']);
+
+        // Test decrement with value 0
+        $inc3 = $this->client->call(Client::METHOD_PATCH, "/databases/$databaseId/collections/$collectionId/documents/$docId/count/increment", array_merge([
```

PHPMD (2.15.0)

Avoid unused local variables such as '$docId'. (Unused Code Rules)

(UnusedLocalVariable)